src/test/java/org/candlepin/subscriptions/tally/admin/InternalTallyResourceTest.java

```diff
@@ -137,4 +191,24 @@ void testPurgeEventRecords() {
     verify(eventRecordRepository)
         .deleteInBulkEventRecordsByTimestampBefore(expectedRetentionTarget);
   }
+
+  ObjectMapper objectMapper(ApplicationProperties applicationProperties) {
```

Comment:
Do we need to configure the object mapper like this for this unit test (and maintain this configuration)? Since we're only asserting if an event is sent, I think that a simple `new ObjectMapper()` would work.

Reply:
We need to define the object mapper because the data does get mapped before the send is called. Otherwise, we get:

com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Java 8 date/time type `java.time.OffsetDateTime` not supported by default: add Module "com.fasterxml.jackson.datatype:jackson-datatype-jsr310" to enable handling